Paralayout/Alignable.swift

```diff
@@ -86,3 +95,69 @@ extension UIView {
     }
 
 }
+
+// MARK: -
+
+public struct FrameAlignmentProxy: Alignable {
+
+    // MARK: - Life Cycle
+
+    public init(proxiedView: UIView) {
+        self.proxiedView = proxiedView
+    }
+
+    // MARK: - Public Properties
+
+    public let proxiedView: UIView
+
+    // MARK: - Alignable
+
+    public var alignmentContext: AlignmentContext {
+        let frameInProxiedViewBounds: CGRect = withViewInSuperview(view: proxiedView) { superview in
+            let frame = proxiedView.frame
+            let originInProxiedViewBounds = try! proxiedView.untransformedConvert(frame.origin, from: superview)
+            let bottomRightInProxiedViewBounds = try! proxiedView.untransformedConvert(
```

Comment:
Is it worth unwrapping these `try!` and providing some meaningful error messages?

Reply:
These methods throw an error when there is no common ancestor between the receiver and the target view. But since we're guaranteeing that the receiver has a superview here, we know that it should never throw (or if it does, it's an internal Paralayout issue). I think it's worth forcing it here, since otherwise it opens up undefined behavior, but maybe this should be a `fatalError` so we can have a better failure message.

This opens up a related question though - right now if you call `align(_:with:_:)` and hit this condition, it will trigger the `ParalayoutAlertForInvalidViewHierarchy` symbol and then no-op. On the other hand, the `alignToSuperview(...)` methods have a precondition that the receiver has a superview. These are both documented behaviors and it isn't necessarily a problem from them to act differently (they're different now because `alignToSuperview` feels more explicit about declaring that the view has a superview), but it might be worth revisiting at some point.